e2e/test-default-route1.sh

```diff
@@ -24,7 +30,13 @@ if [ $ipaddr != "10.1.1.254" ]; then
 fi
 
 echo "check default-route-worker2 interface: net1"
-kubectl exec default-route-worker2 -- ip a show dev net1
+net2=$(kubectl exec default-route-worker2 -- ip a show dev net1)
```

Comment:
the variable `net2` seems not to used anywhere. could you please let me know why `net` is required?

Reply:
ok, echo $net2

Reply:
@s1061123  😅

Reply:
We need kubectl output even though error code is non 0, for troubleshooting. How about to keep `kubectl exec default-route-worker1 -- ip a show dev net1`?

Reply:
This code still seems to be redundant. how about that?

```
kubectl exec default-route-worker2 -- ip a show dev net1
if [ $? -ne 0 ];then
       exit 1
fi
```

Reply:
ok,updated